freshmaker/config.py

```diff
@@ -540,6 +548,11 @@ def _setifok_vcrpy_path(self, s):
             import vcr  # noqa: F401
         self._vcrpy_path = s
 
+    def _setifok_pyxis_server_url(self, s):
```

Comment:
I had forgotten that there is a global config object which means that you can't even import the `freshmaker` package without having a proper configuration set. This just makes things annoying.

My suggestion is to remove this check, set the default value of `pyxis_server_url` and raise an exception in `HandleBotasAdvisory.__init__` if the server URL was not determined.

I'm sorry for missing this earlier.